knowledge-base/tilelayout-responsive.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Responsive TileLayout
+description: How to adjust the size of the TileLayout container or the browser window size changes.
+type: how-to
+page_title: How to make a responsive tilelayout
+slug: tilelayout-kb-responsive
+position: 
+tags: 
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>TileLayout for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I would like to change the dimensions of the TileLayout when the user changes the browser window size.
+
+## Solution
+
+Generally, the `Width` and `Height` parameters of the component can take values in `%`, and the TileLayout will render according to the dimensions of its parent element.
+
+This works well for the initial rendering and the component will be "responsive" immediately according to your layout, regardless of the display (desktop, tablet, phone).
+
+When the layout changes dynamically at runtime, you can use the [TelerikMediaQuery]({%slug mediaquery-overview%}) component to respond to changes of the browser window size. 
```

Comment:
Clarify that this is for other settings, percentage dimensions alone will keep working, but you may want to change others.